### ISSUE TITLE
Talkback navigation is failing 

### ISSUE THREAD

**Issue by dj-4war** [khaleeljageer/LeftNavTV#1]
When the Talkback option is enabled the navigation is inconsistence in-fact not sure where the focus is going. 

Anything to fix this ?

**Comment by raghupatel0** [comment]
Hello Bro Message me in  telegram @Mrhexx

**Comment by shashi-appgolive** [comment]
We can close this issue @raghupatel0 , we have migrated to Compose Navigation. Thanks !

**Comment by manupatell** [comment]
> We can close this issue @raghupatel0 , we have migrated to Compose Navigation. Thanks !

Can You Send Me
